syntax = "proto3";

import "google/protobuf/timestamp.proto";

service UserService {
  rpc CreateUser(CreateUserRequest) returns (CreateUserResponse);
}

message User {
  string id = 1;
  string name = 2;
  google.protobuf.Timestamp create_time = 3;
}

message CreateUserRequest {
  User user = 1;
}

message CreateUserResponse {
  string message = 1;
}